osgi.enroute.web.simple.provider/src/osgi/enroute/web/server/provider/RedirectException.java

```diff
@@ -0,0 +1,14 @@
+package osgi.enroute.web.server.provider;
```

Comment:
By putting the redirect code into a different class, the RedirectException needs to be shared. I thought this was a nicer way to do it, but it could be moved back to being embedded in WebServer.